src/index.js

```diff
@@ -20,8 +20,18 @@ var saveState,
     assetFolder,
     sourceDir; 
 
+function setAppTitle(text) {
+  ipcRenderer.send('setAppTitle', text);
+}
+
+ipcRenderer.on('setAppTitleWithVersion', (event, message) => {
```

Comment:
Are you using this title with version anywhere? It seems like it's mainly just setting the title everywhere using setAppTitle, which doesn't include the version

Reply:
It is a two-way pingpong:
**index.js:** setAppTitle(text) -> `setAppTitle` event
**main.js:** `setAppTitle` event -> add version info -> `setAppTitleWithVersion` event
**index.js:** `setAppTitleWithVersion` event -> update title in the head.

main.js is not aware on the html content,
index.js is not aware on the package info